### ISSUE TITLE
(Sierra 10.12.6) Issue mapping controller inputs

### ISSUE THREAD

**Issue by bongymo** [OpenEmu/OpenEmu#3395]
I've been trying to map an Xbox 360 controller to work with OpenEmu, but every time I try, it only recognizes the button inputs half the time, maps the right analog stick instead of what I'm trying to press, or doesn't map anything at all when I press a button. Never had this issue before when using OpenEmu, not sure what caused it. I've reinstalled OpenEmu and updated my controller drivers, neither of which work. The problem doesn't occur with any other apps, as well. Would appreciate some help on this.

**Comment by clobber** [comment]
Try here first: https://github.com/OpenEmu/OpenEmu/wiki/Troubleshooting:-Input-problems

**Comment by bongymo** [comment]
Tried just about everything on the list, I can't seem to figure out the issue. I did have Enjoy2 installed (hadn't used it in years, though, and even when I had it installed i didn't start having the problem until recently), so I removed that completely. I can usually get it working once after closing and reopening OpenEmu or after a full restart, but after that it goes back to not working most of the time. I don't have any other apps open, either.

Honestly kind of stumped.
